docs/UserGuide.md

```diff
@@ -156,16 +156,65 @@ Format: `exit`
 
 ### Saving the data
 
-AddressBook data are saved in the hard disk automatically after any command that changes the data. There is no need to save manually.
+ContacX data are saved in the hard disk automatically after any command that changes the data in JSON format at `[JAR file location]/data/contacx.json`.
 
+### Exporting the data: `exportcsv`
+
+Exports the current list as a CSV file that can be imported through ContacX or other contact software such as Google/Outlook
```

Comment:
I suggest being more specific for the data being stored
> list *of contacts*
```suggestion
Exports the current list as a CSV file that can be imported through ContaX or other contact software such as Google/Outlook
```

Reply:
amended to fit suggestion, thanks!